settings_test.yml

```diff
@@ -1,7 +1,7 @@
 ---
 default:
   pipename: 'periodic-nightly'
-  users_file: 'tests/static/test_users.yml'
+  users_file: 'users.yml'
```

Comment:
This file is also in ``settings.yml`` also - normal and test settings use these same file with users lists. Tests should use another file, with only tests users.